package.json

```diff
@@ -19,7 +19,7 @@
     "find-business-days-in-range":"1.0.2"
   },
   "engines": {
-      "node": "v6.7.0",
+      "node": "v7.2",
```

Comment:
I would use something like ">=v6" or "v6.x.x" (not tested). This would probably:

* work with CF 245 (contains 6.6.0 and 6.7.0)
* work with CF 250 (contains 6.9.0 and 6.9.1)
* work with future CF releases

See https://docs.npmjs.com/misc/semver